docs/man/libnng.3.adoc

```diff
@@ -455,6 +455,36 @@ with TLS support.
 |xref:nng_tls_config_server_name.3tls.adoc[nng_tls_config_server_name()]|set remote server name
 |===
 
+=== MQTT Support
+
+The library support MQTT V3.1.1 for now (V5.0 is being prepared for development)
```

Comment:
Let's please have an indication that these interfaces are experimental, and may change in future releases as we gain experience with them.

Reply:
will do